packages/components/psammead-radio-schedule/src/StartTime/index.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import styled from 'styled-components';
+import { number, string, shape, oneOf } from 'prop-types';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { C_RHINO, C_PEBBLE } from '@bbc/psammead-styles/colours';
+import { getMinion } from '@bbc/gel-foundations/typography';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { coreIcons } from '@bbc/psammead-assets/svgs';
+import TimestampContainer from '@bbc/psammead-timestamp-container';
+
+const StyledStartTime = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+const StyledClock = styled.span`
+  display: flex;
+  align-items: center;
+  > svg {
```

Comment:
Looking at the designs, the clock shouldn't have a margin to the right, we should reset the default margin on the `svg` and apply some padding to the StyledClock.

Reply:
Good point, done

Reply:
Sorry, looking at the designs I can't seem to find where `const CLOCK_SIDE_PADDING = '0.51875rem'` is coming from

Reply:
<img width="545" alt="Screenshot 2020-01-24 at 10 43 57" src="https://user-images.githubusercontent.com/17802955/73063339-7ceaca80-3e96-11ea-96aa-3e286ef38870.png">

This 8.3px as far as I understand correctly

Reply:
That value also seemed a bit strange to me, I will double check with UX that it's supposed to be 8.3px not 8px

Reply:
Okay so Laura said that 8.3 was a Zeplin glitch so I changed the padding value to 8px